.github/workflows/build.yml

```diff
@@ -317,3 +320,44 @@ jobs:
       run: make pythoninfo
     - name: Tests
       run: xvfb-run make buildbottest TESTOPTS="-j4 -uall,-cpu"
+
+  check:  # This job does nothing and is only used for the branch protection
+    if: always()
```

Comment:
This is necessary because otherwise it'll get a `skipped` status in some cases and the branch protection sees that as `skipped == success` which is undesired.